libs/api/ebpf_api.cpp

```diff
@@ -412,8 +412,14 @@ _ebpf_map_lookup_element_helper(fd_t map_fd, bool find_and_delete, _In_opt_ cons
         goto Exit;
     }
     assert(value_size != 0);
+    if (type == BPF_MAP_TYPE_PERCPU_ARRAY || type == BPF_MAP_TYPE_PERCPU_HASH || type == BPF_MAP_TYPE_LRU_PERCPU_HASH) {
```

Comment:
not required but nice to have:
would be nice to be able to get a per_cpu boolean from _get_map_descriptor_properties rather than relying on an explicit list of types here, especially if we add another per-cpu type in the future where we might forget to update this line.   ebpf_maps.c already stores a per_cpu boolean per map type.

Reply:
OK, this ends up being a lot more work than expected. To solve the original problem, I will just add a central macro to test for the per-cpu property (so there is only one location to update).